src/commcare_cloud/ansible/roles/datadog/tasks/main.yml

```diff
@@ -3,6 +3,11 @@
   tags:
     - datadog
 
+- name: Add datadog apt key
+  apt_key: id=33EE313BAD9589B7 keyserver=keyserver.ubuntu.com state=present
```

Comment:
Can you link to the documentation where this comes from?

Reply:
@dannyroberts I couldn't find the right link which actually pointed me to replace the key with the latest one. 
But I picked the RSA from this location 
1. http://keyserver.ubuntu.com/ -- searched for datadog. 
2. got routed to this page http://keyserver.ubuntu.com/pks/lookup?search=datadog&fingerprint=on&op=index
3. went to this link from response page http://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0x32637d44f14f620e
4. Add tested on a server.

Reply:
going forward will make sure with links.